website/docs/cluster-management/managing-clusters-without-capi.mdx

```diff
@@ -8,31 +8,31 @@ import TabItem from "@theme/TabItem";
 
 import TierLabel from "../_components/TierLabel";
 
-# Managing existing clusters <TierLabel tiers="Enterprise" />
+# Managing Clusters Without Cluster API <TierLabel tiers="Enterprise" />
 
-### Managing non-capi clusters {#how-to-connect-a-cluster}
-
-Any kubernetes cluster whether capi or not can be added to Weave Gitops Enterprise. The only thing we need is a secret containing a valid `kubeconfig`.
+You do **not** need Cluster API to add your Kubernetes cluster to Weave Gitops Enterprise. The only thing you need is a secret containing a valid `kubeconfig`.
 
 import TOCInline from "@theme/TOCInline";
 <TOCInline />;
 
 <Tabs groupId="have-a-secret" default>
 <TabItem value="existing-kubeconfig-secret" label="Existing kubeconfig">
 
-If you already have a `kubeconfig` stored in a secret in your management cluster, continue below to create a `GitopsCluster`.
+## Adding kubeconfig to Your Management Cluster
```

Comment:
Not sure if this title was meant to be within the `Existing kubeconfig` tab specifically or whether it is meant to be a title for both tabs.

Reply:
It's a title for the tab. It does look a little "off" but I think it's because we probably should make the tabs more distinct than they are.